pkg/command/plugin.go

```diff
@@ -528,18 +528,24 @@ func getInstalledAndMissingContextPlugins() (installed, missing []discovery.Disc
 
 			// Store the installed plugin, which includes the context from which it was installed
 			found = true
+			if serverPlugins[i].RecommendedVersion != installedPlugins[j].Version {
+				serverPlugins[i].Status = common.PluginStatusUpdateAvailable
+				pluginSyncRequired = true
+			} else {
+				serverPlugins[i].Status = common.PluginStatusInstalled
+			}
 			serverPlugins[i].InstalledVersion = installedPlugins[j].Version
```

Comment:
When this happens, should we be should `RecommendedVersion` or the `InstalledVersion` of the plugin in the `VERSION` column? cc @vuil @prkalle 

This change continues to show the `InstalledVersion` of the plugin along with the `updates available` status.

Reply:
I'd say it is more consistent to show the installed version. 

Nit: (feel free to ignore, as I am not sure it is worth tweaking). 'Updates available' suggests to me there is a newer version of the plugin to update to, but the logic allows for syncing down to an older version too.

Reply:
> This change continues to show the InstalledVersion of the plugin along with the updates available status.

This sound good to me

Reply:
I do not have better words to describe that state so may be we can go with the existing `updates available` that we used with the legacy CLI and update it in the future if we thing necessary.

Reply:
Sounds good to me, but it may be confusing to user and it may prompt user to keep trying plugin sync by seeing `updates available` in the status, may be we should document it.